sdk/identity/identity/src/credentials/clientAssertionCredential.browser.ts

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { AccessToken, TokenCredential } from "@azure/core-auth";
+import { credentialLogger, formatError } from "../util/logging";
+
+const BrowserNotSupportedError = new Error(
+  "ClientAssertionCredential is not supported in the browser."
+);
+const logger = credentialLogger("ClientAssertionCredential");
```

Comment:
Not sure logging is required here since it's just throwing.

Reply:
We do logging everywhere, since this might be useful for the chained credentials like - DefaultAzureCredentials and also in cases where the inner error messages or errors thrown might get suppressed due to some reason.

Reply:
Ok, just comparing against other classes that do implement for browsers don't log

Reply:
For identity, we have logged for all credentials that don't support browsers